class1.md

```diff
@@ -0,0 +1,261 @@
+
+Biology is suited for machine learning applications:
+* You cannot code the rules
+    * Biological systems are incredibly complex with many inputs leading to complex phenotypes. 
+    * Example: if you want to determine whether or not a patient is likely to get cancer we’d need to take many datasets into account (demographic data, health/nutrition data, family history, genetic data, etc). We cannot determine and code the rules for a computer to calculate if a person is likely to get cancer or not.
+
+* You cannot scale
+    * Our human brains cannot make sense of these huge integrated datasets or even just one of them.
+    * Example: a human looking at a series of bases would take years to make sense of the human genome. A computer can find patterns in our genome with relative ease.
+
+### How is machine learning applied?
+
+<img align="right" width="150" src="https://media.giphy.com/media/3oEduS0HyWuqR7kXL2/giphy.gif">
+
+Machine learning applications are all around us! Every time Spotify serves up a customized playlist or radio station based on the music that you like, Facebook loads your newsfeed, your phone autocorrects you, or you interact with a voice assitant like Siri, you are interacting with machine learning algorithms.
+
+Every-day applications:
+* Search engines
+* Recommendation systems (Yelp, Netflix, Spotify)
+* Voice assistants
+* Social media feeds
+* Text prediction
+
+#### Machine learning techniques are becoming increasingly important to biological research.
+
+Researchers are developing algorithms that use machine learning to classify histopathological images, visualize and find clusters within single cell datasets, and much more.
+
+Biology specific applications:
+* Prediction of protein secondary structure based on RNA-sequence.
+* Classification of histopathological images of breast tissue to identify malignant tumors.
+* Using gene expression data to classify patients into different clinical groups.
+
+Machine learning work that is happening here at Fred Hutch:
+* Gottardo lab - [automatic gating for flow cytometry data.](https://www.biorxiv.org/content/10.1101/702118v2)
+* Emily Silgard's data science group - [natural language processing to extract information from unstructured electronic medical records notes.](https://www.sciencedirect.com/science/article/abs/pii/S1532046419302217)
+* The Data Visualization Center - [single-cell visualization and clustering of RNA-sequencing data to find rare subtypes of cells that emerge in the mouse embryo.](https://oncoscape.v3.sttrcancer.org/atlas.gs.washington.edu.mouse.rna/landing)
+
+### How do these machine learning applications work?
+
+There are different types of machine learning and each have their own specific applications and requirements.
+
+Four major paradigms in machine learning:
```

Comment:
I'd include brief definitions for each of these here, especially since you don't mention the last two at all anywhere else